rero_ils/modules/cli.py

```diff
@@ -225,3 +233,181 @@ def create(infile, pid_type, schema, verbose, dbcommit, reindex):
+                            extension=extension
+                        )),
+                        recursive=True
+                    )
+        return files_list
+
+    def get_files_flat(paths, extensions):
+        """Get files from paths."""
+        files_list = []
+        for path in paths:
+            if os.path.isdir(path):
+                for extension in extensions:
+                    files_list += glob(
+                        os.path.join(path, '*.{extension}'.format(
+                            extension=extension
+                        )),
+                        recursive=False
+                    )
+        return files_list
+
+    def delete_prefix(prefix, line):
+        """Delete prefix from line."""
+        if prefix:
+            line = line.replace(prefix, "")
+        return line.strip()
+
+    def is_copyright(copyrides, line):
+        """Delete prefix from line."""
+        for copyride in copyrides[1:]:
+            if line == copyride:
+                return True
+        return False
+
+    def show_diff(linenbr, text, n_text):
+        """Show string diffs."""
+        seqm = difflib.SequenceMatcher(
+            None,
+            text.replace(' ', '◼︎'),
+            n_text.replace(' ', '◼︎')
+        )
+        click.echo('{linenbr}: '.format(linenbr=linenbr), nl=False)
+        for opcode, a0, a1, b0, b1 in seqm.get_opcodes():
+            if opcode == 'equal':
+                click.echo(seqm.a[a0:a1], nl=False)
+            elif opcode == 'insert':
+                click.secho(seqm.b[b0:b1], fg='red', nl=False)
+            elif opcode == 'delete':
+                click.secho(seqm.a[a0:a1], fg='blue', nl=False)
+            elif opcode == 'replace':
+                # seqm.a[a0:a1] -> seqm.b[b0:b1]
+                click.secho(seqm.b[b0:b1], fg='green', nl=False)
+        click.echo()
+
+    def test_file(file_name, extension, copyrights, license_lines, verbose,
+                  progress):
+        """Test the license in file."""
+        if progress:
+            click.secho('License test: ', fg='green', nl=False)
+            click.echo('{file_name}'.format(file_name=file_name))
+        with open(file_name, 'r') as file:
+            result = test_license(
+                file=file,
+                extension=config['file_extensions'][extension],
+                copyrights=config['copyrights'],
+                license_lines=license_lines,
+                verbose=verbose
+            )
+            if result != []:
+                click.secho(
+                    'License error in {file} in lines {lines}'.format(
+                        file=file_name,
+                        lines=result
+                    ),
+                    fg='red'
+                )
+                return 1
+        return 0
+
+    def test_license(file, extension, copyrights, license_lines, verbose):
+        """Test the license in file."""
```

Comment:
please change it as it is the same as previous

Reply:
ok